### ISSUE TITLE
Make a non subbed version(if its possible)

### ISSUE THREAD

**Issue by HTaamas** [N0-0NE-Dev/NoFasel#43]

**Comment by N0-0NE-Dev** [comment]
Yeah, for a while HDwatched was working, but seems like it's not anymore. I will look into a different provider.

**Comment by DarkCrypt** [comment]
@N0-0NE-Dev 
There's an endless supply of sites to scrape obviously but I'd use the sites within the list of [Cloudstream](https://codeberg.org/cloudstream) repos on Codeberg.

**Comment by Uvdon** [comment]
Please add movie-web.app provider

**Comment by NazzarenoGiannelli** [comment]
The few movies I tried for now are looking and sounding great! It would be amazing to have no arabic subs if possible. Everything else is 👌
